tests/wpt/web-platform-tests/fullscreen/model/remove-single.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<title>Remove the single element on the fullscreen element stack</title>
```

Comment:
I don't think this test will pass in other browsers because of https://fullscreen.spec.whatwg.org/#allowed-to-request-fullscreen . We should add this test to tests/wpt/mozilla/sts/mozilla/ instead.